076-support-for-kafka-4.0.md

```diff
@@ -0,0 +1,203 @@
+# Support for Apache Kafka 4.0
+
+Apache Kafka is planning a new major release - Apache Kafka 4.0.
+Unlike previous major releases that Strimzi handled before (2.0 and 3.0), this one is "special" because it drops the support for ZooKeeper-based clusters.
```

Comment:
```suggestion
Unlike previous major releases that Strimzi handled before (2.x and 3.x), this one is "special" because it drops the support for ZooKeeper-based clusters.
```